main.go

```diff
@@ -422,6 +422,13 @@ func main() {
 
 	RegisterHTTPReader(k)
 	go func() {
+		url := "http://"
+		if strings.HasPrefix(*listen, ":") {
+			url += "localhost"
+		}
+		url += *listen + "/"
+
+		log.Println("Starting Webserver on ", url)
```

Comment:
nvm, just realized. I would still rather do a `if (*listen)[0] == ':' { *listen = "localhost" + *listen }` and then the Printf